docs/coding.rst

```diff
@@ -664,7 +746,13 @@ Picto
 
 
 Call out
-~~~~~~~~
+<<<<<<<<
+
+- macro import
```

Comment:
**suggestion (non-blocking):** visually, I don't see a clear distinction between imports and parameters

<img width="599" alt="Screen Shot 2022-01-04 at 3 53 47 PM" src="https://user-images.githubusercontent.com/19650432/148086348-6609001c-c50d-44ac-8989-643a6f348743.png">

Maybe we could separate them out a bit and take advantage of the code block clipboard functionality?

```suggestion
**HTML**

.. code-block:: html

    {% from "macros-protocol.html" import picto with context %}

**SCSS**

.. code-block:: css

    @import '~@mozilla-protocol/core/protocol/css/components/picto';

**Params**

- title
```
i.e.
<img width="712" alt="Screen Shot 2022-01-04 at 3 49 26 PM" src="https://user-images.githubusercontent.com/19650432/148086024-18202205-1e9a-4d40-a508-60d9c13dde29.png">

Reply:
Good suggestion! :)